File: Fuentes/Principal/Logica/Usuarios.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Logica
{
    public class Usuarios
    {
         
        public static int id;
        public static string nombre;
        public static string contrasena;
        public static int nivel;
        public static bool accesoTotal;

        public static void ObtenerParametros()
        {

            string[] parametros = Environment.GetCommandLineArgs().ToArray(); 
            if (parametros.Length > 1)
            {
                int numeracion = 10;
                id = Funciones.ValidarNumero(parametros[numeracion].Replace("|", " ")); numeracion += 1; 
            }

        }

    }
}

File: Fuentes/Salidas/Logica/Usuarios.vb
Public Class Usuarios

    Public Shared id As Integer
    Public Shared nombre As String
    Public Shared contrasena As String
    Public Shared nivel As Integer
    Public Shared accesoTotal As Boolean

    Public Shared Sub ObtenerParametros()

        Dim parametros() = Environment.GetCommandLineArgs().ToArray() 
        If (parametros.Length > 0) Then
            Dim numeracion As Integer = 10
            id = Convert.ToInt32(parametros(numeracion).Replace("|", " ")) : numeracion += 1
        End If

    End Sub

End Class
